### PR TITLE
add Costco Food Court

### PR DESCRIPTION
**Pull request by modaserMoj**

### CODE REVIEW

File: data/brands/amenity/fast_food.json [hunk 1]
```diff
@@ -908,6 +908,19 @@
         "takeaway": "yes"
       }
     },
+    {
+      "displayName": "Costco Food Court",
+      "id": "costcofoodcourt-4d2ff4",
+      "locationSet": {"include": ["us"]},
```
Comment (tas50) [inline data/brands/amenity/fast_food.json:914]: Do the worldwide costcos have the food court?

Comment (modaserMoj) [reply data/brands/amenity/fast_food.json:914]: I can add a few of the countries that I found but I can find some sort of a complete list. Should I add those countries?

Comment (tas50) [reply data/brands/amenity/fast_food.json:914]: Add whatever you can find and folks can build on it from there.

Comment (modaserMoj) [reply data/brands/amenity/fast_food.json]: I added what I could find

**Comment by tas50** [comment]
Thanks for the update

**Pull request merged**